nginx/check.py

```diff
@@ -44,13 +44,26 @@ def check(self, instance):
         else:
             metrics = self.parse_text(response, tags)
 
+        UPSTREAM_RESPONSE_CODES = [
```

Comment:
let's move this constant to the class body or the file body to define only once.

also, we may want to rename the constant to something that reflects that these metrics will be sent as `count` on top of their "original" metric type/name. Maybe `SEND_AS_COUNT`?  Anyway let me know what you think